pkg/inclusion/paths_test.go

```diff
@@ -338,97 +340,144 @@ func Test_genSubTreeRootPath(t *testing.T) {
 
 func Test_calculateCommitPaths(t *testing.T) {
 	type test struct {
-		squareSize int
-		start      int
-		blobLen    int
-		expected   []path
+		name                string
+		squareSize          int
+		start               int
+		blobLen             int
+		expectedPath        []path
+		expectedPathIndexes []int
 	}
+	// note that calculateCommitPaths assumes ODS, so we prepend a WalkLeft to
+	// everything elsewhere
 	tests := []test{
-		{2, 0, 1, []path{{instructions: []WalkInstruction{WalkLeft}, row: 0}}},
-		{2, 2, 2, []path{{instructions: []WalkInstruction{}, row: 1}}},
```

Comment:
had to re-write these tests cause now there are way too many damn paths to type by hand. Now this test only looks for specific paths of the result.